code/modules/food_and_drinks/recipes/tablecraft/recipes_lizard.dm

```diff
@@ -187,6 +187,17 @@
 	result = /obj/item/food/rootdough
 	category = CAT_LIZARD
 
+/datum/crafting_recipe/food/rootdough2
+	name = "Rootdough"
+	reqs = list(
+		/obj/item/food/grown/potato = 2,
+		/datum/reagent/consumable/soymilk = 15,
+		/datum/reagent/consumable/korta_flour = 5,
+		/datum/reagent/water = 10
+	)
+	result = /obj/item/food/rootdough
+	category = CAT_LIZARD
+
```

Comment:
This change is not noted either in the PR body or change notes, and resultingly also isn't justified by any of the explanatory text. Did you include it by mistake?

Reply:
It's listed in there, it's also listed in the screenshot.

Reply:
"This PR has a few simple aims; 2 new rootbread sandwiches, slicable versions of most pies and a **new rootbread crafting method**."

Reply:
I will add it to the changelog shortly when I return to my PC.

Reply:
Please also add why it is good for the game.

Reply:
Done.